mix.exs

```diff
@@ -9,25 +9,24 @@ defmodule Recaptcha.Mixfile do
       description: description(),
       deps: deps(),
       package: package(),
-
-      build_embedded: Mix.env == :prod,
-      start_permanent: Mix.env == :prod,
+      build_embedded: Mix.env() == :prod,
```

Comment:
Can you please update `:elixir_version` value as well? 
As we do not run tests in `1.2` anymore.

Reply:
Yeah, no problem. I'll update this to 1.4